src/element.ohlc.js

```diff
@@ -0,0 +1,52 @@
+﻿'use strict';
+
+module.exports = function(Chart) {
+
+	var helpers = Chart.helpers;
+	var globalOpts = Chart.defaults.global;
+	var defaultColor = globalOpts.defaultColor;
+
+	globalOpts.elements.ohlc = {
+		lineWidth: 2,
+		armLength: 6,
+		color: {
+			up: globalOpts.elements.candlestick.upCandleColor,
+			linear: '#888',
+			down: globalOpts.elements.candlestick.downCandleColor,
+		}
+	};
+	
+	Chart.elements.ohlc = Chart.elements.Candlestick.extend({
```

Comment:
we probably should have a `Financial` that both `Candlestick` and `OHLC` extend from. ditto for the controllers

Reply:
I renamed the `financial` controller to  `candlestick` (so it's more consistent between the controller and the element)
For now `ohlc` extends `candlestick` (both for element and controller).
Should we create the new `financial` controller and element before merging this pull request ?

Reply:
Yes, can you create a new `financial` controller and element?

Reply:
Done
But I'm not 100% sure what should be in `financial` and what should be in its children.